{"name": "test_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'App' object has no attribute 'get_screenshot_as_png'", "trace": "tp = <class 'AttributeError'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n                raise value.with_traceback(tb)\n>           raise value\n\n/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/six.py:693: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/six.py:693: in reraise\n    raise value\n/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/six.py:693: in reraise\n    raise value\n/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/allure_commons/_allure.py:169: in __call__\n    return self._fixture_function(*args, **kwargs)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def fin():\n        driver = App(base_url = base_url)\n>       attach = driver.get_screenshot_as_png()\nE       AttributeError: 'App' object has no attribute 'get_screenshot_as_png'\n\n../conftest.py:38: AttributeError"}, "attachments": [{"name": "stdout", "source": "803f7f1b-8dbc-430b-a3cd-8fb004b89759-attachment.txt", "type": "text/plain"}], "start": 1559919348624, "stop": 1559919348625, "uuid": "2e175c1a-8461-40db-89f7-1456dc04d663", "historyId": "1d6c5e711b29cf7bad43b6e24d6adc90", "fullName": "tests.test#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "Work-Mac-Mini.local"}, {"name": "thread", "value": "90494-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test"}]}